### ISSUE TITLE
`requirements.yml` naming

### ISSUE THREAD

**Issue by betatim** [CUB-Computational-Tools/assignment_problem_set#2]
Came here from your comment in: https://github.com/jupyterhub/binder/issues/88#issuecomment-440556310

Are you sure `requirements.yml` gets used ever? It should be called `environment.yml` I think.

**Comment by betatim** [comment]
Related to this, do you have to install jupyter lab "by hand" in the postBuild? I was expecting that it would be installed by default?

**Comment by sebkopf** [comment]
thanks @betatim, that solved my problem! when I switched from `requirements.txt` I completely got the conda config file name wrong. I thought it had to do with the `install.R` instructions. Works beautifully now with just the `environment.yml` python dependencies and `install.R` R packages. Jupyter lab is installed by default it seems, no need to add it in postBuild, or do you mean the `jupyterlab/toc` labextension should also be installed by default already?

**Comment by betatim** [comment]
Perfect! Do you want to make a super small "barebones" example for binder-examples on having R and Python installed at the same time and happily co-existing?

**Comment by sebkopf** [comment]
sure thing, here is the barebones example: https://github.com/KopfLab/R_python_binder_example

I updated the link in https://github.com/jupyterhub/binder/issues/88#issuecomment-440556310, is there anywhere else I should post this?

**Comment by betatim** [comment]
@choldgraf thoughts on moving the repo linked in the above comment to `binder-examples`?

**Comment by choldgraf** [comment]
Sounds great to me - I like the idea of showing how this is done w/o purely using conda

**Comment by sebkopf** [comment]
@betatim @choldgraf sounds great, should I just transfer owernship to the `binder-examples` organization?

**Comment by betatim** [comment]
I sent you an invite to the binder-examples org that should give you the privileges needed for the transfer.

When you transfer the repo can you rename it a bit? we could remove the "example" and "binder" as it should be clear from the org :) However just R_Python is a bit weird? I'll think if I can make a concrete suggestion, otherwise just go with what you think works well.

Maybe in a second step we can check how much overlap there is between the docs in the readme of https://github.com/binder-examples/r and R_python_binder_example. We should make sure that where they give advice on how to do the same thing they don't disagree :)

**Comment by choldgraf** [comment]
@betatim I agree - we should open an issue to consider merging (or at least referencing) these two repositories once this one is in there.

@sebkopf feel free to ping here once the repo is transferred, and *THANKS* for helping out :-)

**Comment by sebkopf** [comment]
@betatim I transfered to binder-examples org, renamed to `r_with_python`

@choldgraf I added a link to the standard r binder example but I agree that there is a fair bit of overlap that might be worth merging as you see fit. The ipython notebooks are a fairly trivial extension of the r binder although it may be worth pointing out to users how easy it is to run both (I did not realize at first just how easy). So is the RMarkdown file using R of course. The more interesting addition I think is the use of reticulate to run python from an RMarkdown file although it won't be available to its fullest potential until the default RStudio installation is v 1.2+ (whenever that's officially released and becomes the default).
